src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos, getUser } from './api';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | null>(null);
+  const [isTodosLoaded, setIsTodosLoaded] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [isButtonClicked, setIsButtonClicked] = useState(false);
+  const [selectedOption, setSelectedOption] = useState('all');
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    getTodos().then(visTodos => {
+      setTodos(visTodos);
+      setIsTodosLoaded(true);
```

Comment:
```suggestion
    getTodos()
      .then(visTodos => {
        setTodos(visTodos);
        setIsTodosLoaded(true);
```
I'm not sure, but `.then` should be written in a new line...but I can be mistaken. In any case, as for me, it is more understandable and clear